### PR TITLE
Database setup

### PR DESCRIPTION
**Pull request by Mr-emeka**
I've set up the database environment for Production and Development.

### CODE REVIEW

**Comment by Mr-emeka** [comment]
@Mohamilr dev database is the one on my machine while the Production database is the one on Heroku.

**Comment by Mr-emeka** [comment]
@Rexben001 I'm having an error on Travis something to do with my database though it's passing.

**Comment by Rexben001** [comment]
> @Rexben001 I'm having an error on Travis something to do with my database though it's passing.

You need to create a db for your test on Travis

**Comment by Mr-emeka** [comment]
How do I go about it @Rexben001 I'm pretty much clueless

File: src/models/db.js [hunk 1]
```diff
@@ -1,17 +1,37 @@
 import pg from 'pg';
+import {
+    parse
+  } from 'pg-connection-string';
 import dotenv from 'dotenv';
 
 dotenv.config();
```
Comment (Rexben001) [inline src/models/db.js:8]: I can't see your test db, you only have for dev and production

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+DEV_USER = postgres
+DEV_PASSWORD= 
+DEV_SERVER=localhost
+DEV_PORT=5432
+DEV_DATABASE=reactjs
+
+DATABASE_URL = postgres://{db_username}:{db_password}@{host}:{port}/{db_name}
+
+
+DB_USER = ruazooloskcdhy
```
Comment (Rexben001) [inline .env.example]: This credential is public, it should be private

**Review by Rexben001 — CHANGES_REQUESTED** [review]

File: package.json [hunk 3]
```diff
@@ -27,13 +27,14 @@
     "@babel/polyfill": "^7.6.0",
```
Comment (Rexben001) [inline package.json:30]: How do you manage to change the env to dev or test?
You can install `cross_env`
Your test script can look like `cross-env NODE_ENV=test nyc mocha...`

**Review by Rexben001 — APPROVED** [review]

**Pull request merged**